<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-80</x>
      <y>-70</y>
      <width>881</width>
      <height>701</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="draw">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>630</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>draw</string>
    </property>
   </widget>
   <widget class="QFrame" name="commonMethods">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>421</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="moveUp">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>moveUp</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveLeft">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>moveLeft</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveDown">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>moveDown</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveRight">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>moveRight</string>
     </property>
    </widget>
    <widget class="QPushButton" name="decrease">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>decrease</string>
     </property>
    </widget>
    <widget class="QPushButton" name="increase">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>9</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>increase</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateRight">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>rotateRight</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rotateLeft">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>0</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>rotateLeft</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectCenter">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>120</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectCenter">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>disconnect</string>
     </property>
    </widget>
    <widget class="QFrame" name="block">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>181</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="uniqueMethods">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>421</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>241</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="circlePage"/>
     <widget class="QWidget" name="RectanglePage">
      <widget class="QDoubleSpinBox" name="heightRectangle">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>130</y>
         <width>141</width>
         <height>91</height>
        </rect>
       </property>
       <property name="minimum">
        <double>10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>500.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>3.000000000000000</double>
       </property>
       <property name="value">
        <double>200.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="widthRectangle">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>40</y>
         <width>141</width>
         <height>91</height>
        </rect>
       </property>
       <property name="minimum">
        <double>10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>500.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>3.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="RectangleWidth">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>46</y>
         <width>121</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Rectangle width</string>
       </property>
      </widget>
      <widget class="QLabel" name="RectangleHeight">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>121</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string>Rectangle height</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="TrianglePage">
      <widget class="QDoubleSpinBox" name="valueAside">
       <property name="geometry">
        <rect>
         <x>231</x>
         <y>20</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="minimum">
        <double>100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>5.000000000000000</double>
       </property>
       <property name="value">
        <double>200.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="valueBside">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>90</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="minimum">
        <double>100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>5.000000000000000</double>
       </property>
       <property name="value">
        <double>200.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="valueCside">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>160</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="minimum">
        <double>100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>5.000000000000000</double>
       </property>
       <property name="value">
        <double>200.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="sideA">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>26</y>
         <width>81</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;sideA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="sideB">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>81</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;sideB&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="sideC">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>160</y>
         <width>81</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;sideC&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="RhombPage">
      <widget class="QDoubleSpinBox" name="setRhombWidth">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>161</width>
         <height>101</height>
        </rect>
       </property>
       <property name="maximum">
        <double>500.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>3.000000000000000</double>
       </property>
       <property name="value">
        <double>99.989999999999995</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="setRhombHeight">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>120</y>
         <width>161</width>
         <height>111</height>
        </rect>
       </property>
       <property name="maximum">
        <double>500.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>3.000000000000000</double>
       </property>
       <property name="value">
        <double>200.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="RhombWidth">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>151</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RhombWidth</string>
       </property>
      </widget>
      <widget class="QLabel" name="RhombHeight">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>141</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RhombHeight</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="StarPage">
      <widget class="QPushButton" name="innerRadiusDown">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>innerRadiusDown</string>
       </property>
      </widget>
      <widget class="QPushButton" name="innerRadiusUp">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>innerRadiusUp</string>
       </property>
      </widget>
      <widget class="QPushButton" name="outerRadiusDown">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>outerRadiusDown</string>
       </property>
      </widget>
      <widget class="QPushButton" name="outerRadiusUp">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>80</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>outerRadiusUp</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="vertexes">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>161</width>
         <height>91</height>
        </rect>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>146</y>
         <width>151</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Star Vertexes</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page">
      <widget class="QSpinBox" name="spinBoxPolygon">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>131</width>
         <height>71</height>
        </rect>
       </property>
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="value">
        <number>6</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>131</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Vertixes</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="ChangePolyline">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>ChangePolyline</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>411</width>
       <height>191</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>371</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="textCenter">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textSquare">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textPerimeter">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>721</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="polyline">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>polyline</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rhombus">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>rhombus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="triangle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>triangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Hexagon">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Hexagon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rectangle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>rectangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="circle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>circle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Star">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>star</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
